### DISCUSSION TITLE
Release 2.17.1 with intention of 2.17.2 later

### DISCUSSION THREAD

**Discussion by hazendaz** [revelc/formatter-maven-plugin#541] [General]
@ctubbsii I think we should probably just go ahead and release 2.17.1 cut.  While it doesn't have a lot in it, waiting on jsoup to release seems slow and I would like to get a few cycles on new logic around line ending normalization changes.  Thoughts?

**Comment by ctubbsii** [comment]
@hazendaz Sure. I could do it this weekend, Monday at the latest, unless you want to do it. Are you confident the line ending stuff is working the way you think it should be working now?

**Comment by hazendaz** [comment]
@ctubbsii I'm good with what is here, I was hoping to get jsoup updates but given how long its been, I'd rather get a release out so the tires can be kicked a bit before we do this again anyways in December with 2021-12.  If you have time that would be great.

**Comment by ctubbsii** [comment]
Released 2.17.1

**Comment by hazendaz** [comment]
I do still want to try to get a 2.17.2 out especially around issue with how we are handing the cache.  I presume impsort has similar issue.  That one looks similar and not designed for automated bot processing in either case.  So I will be working through week here.  I'm trying to get jsoup team to release something with there code.  If I get no response from them, I'm going to just release a fork so we get off the snapshot.

**Reply by ctubbsii** [reply]
We've already bumped the Eclipse jdt.core version, so it would make more sense to release 2.18.0 for that. I don't see any reason to backport a cache fix to 2.17, since it already requires Java 11 anyway. It might make sense to backport a cache fix to 2.16, which only requires Java 8, if you really want to, though (but I wouldn't bother).

**Comment by hazendaz** [comment]
ok agreed.  Our numbering scheme doesn't really align with Eclipse so it doesn't much matter and this jsoup work being as extensive and pain makes a minor jump not that ideal.

**Comment by hazendaz** [comment]
agreed no 2.17.2, will go to 2.18.0, archiving discussion.
